cis_proc.py

```diff
@@ -185,14 +176,14 @@ def main(argv=None):
         dset_files = ['CHANGES', 'README', 'dataset_description.json',
                       'participants.tsv']
         for dset_file in dset_files:
-            if not op.isfile(op.join(args.bids_dir, dset_file)):
+            if not op.isfile(op.join(bids_dir, dset_file)):
                 shutil.copyfile(op.join(scan_work_dir, 'bids', dset_file),
-                                op.join(args.bids_dir, dset_file))
+                                op.join(bids_dir, dset_file))
 
         p_df = pd.read_csv(op.join(scan_work_dir, 'bids/participants.tsv'),
                            sep='\t')
         p_df = p_df.T.drop_duplicates().T
-        p_df2 = pd.read_csv(op.join(args.bids_dir, 'participants.tsv'),
+        p_df2 = pd.read_csv(op.join(bids_dir, 'participants.tsv'),
                             sep='\t')
         p_df2 = p_df2.T.drop_duplicates().T
```

Comment:
I think we should work on naming for these temporary dataframes

Reply:
Now there are more informative names.